src/Compilers/VisualBasic/Portable/BoundTree/Expression.vb

```diff
@@ -2384,11 +2465,17 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
         End Function
     End Class
 
-    Friend Partial Class BoundLateBoundArgumentSupportingAssignmentWithCapture
+    Partial Friend Class BoundLateBoundArgumentSupportingAssignmentWithCapture
         Protected Overrides Function ExpressionKind() As OperationKind
             Return OperationKind.None
         End Function
 
+        Protected Overrides ReadOnly Property Children As ImmutableArray(Of IOperation)
```

Comment:
>Protected Overrides ReadOnly Property Children As ImmutableArray(Of IOperation) [](start = 8, length = 79)

BoundLateBoundArgumentSupportingAssignmentWithCapture looks like a lower node, no need to modify it.